yt/fields/tests/test_magnetic_fields.py

```diff
@@ -19,19 +19,26 @@ def test_magnetic_fields():
 
     ds1 = load_uniform_grid(data1, ddims, unit_system="cgs")
     ds2 = load_uniform_grid(data2, ddims, unit_system="mks")
+    # By default, code units are cgs units
```

Comment:
I think code units are only cgs units for gizmo, but given that this is only implemented for gizmo, I guess that is OK?

Reply:
Actually, I'm not very familiar with how the code units are determined in general. I checked that for this test dataset, it's actually cgs. I'll make the comments more specific.